bio_hansel/subtyping_params.py

```diff
@@ -3,12 +3,12 @@
 
 @attr.s
 class SubtypingParams(object):
-    low_coverage_depth_freq = attr.ib(default=20, validator=attr.validators.instance_of(int))
+    low_coverage_depth_freq = attr.ib(default=20.0, validator=attr.validators.instance_of((float, int)))
     max_perc_missing_tiles = attr.ib(default=0.05, validator=attr.validators.instance_of(float))
     min_ambiguous_tiles = attr.ib(default=3, validator=attr.validators.instance_of(int))
     max_perc_intermediate_tiles = attr.ib(default=0.05, validator=attr.validators.instance_of(float))
-    min_kmer_freq = attr.ib(default=10, validator=attr.validators.instance_of(int))
-    max_kmer_freq = attr.ib(default=200, validator=attr.validators.instance_of(int))
+    min_kmer_freq = attr.ib(default=10, validator=attr.validators.instance_of((float, int)))
+    max_kmer_freq = attr.ib(default=200, validator=attr.validators.instance_of((float, int)))
```

Comment:
`max_kmer_freq` default should be `1000`. #18

Reply:
Fixed #18 in a920b4f0a26586249fe960b1909e552ee3c1b152